blosc/CMakeLists.txt

```diff
@@ -185,13 +185,22 @@ if(COMPILER_SUPPORT_SSE2)
                     SOURCE shuffle.c
                     APPEND PROPERTY COMPILE_OPTIONS "/arch:SSE2")
         endif()
+        # Add SIMD flags for the bytedelta filter and Intel (it seems that ARM64 does not need these)
+        # https://stackoverflow.com/questions/64053597/how-do-i-enable-sse4-1-and-sse3-but-not-avx-in-msvc
+        set_source_files_properties(
+                ${PROJECT_SOURCE_DIR}/plugins/filters/bytedelta/bytedelta.c
+                PROPERTIES COMPILE_OPTIONS "/d2archSSE42")
     else()
         set_source_files_properties(
                 shuffle-sse2.c bitshuffle-sse2.c blosclz.c fastcopy.c
                 PROPERTIES COMPILE_FLAGS -msse2)
         set_property(
                 SOURCE shuffle.c
                 APPEND PROPERTY COMPILE_OPTIONS -msse2)
+        # Add SIMD flags for the bytedelta filter and Intel (it seems that ARM64 does not need these)
+        set_source_files_properties(
+                ${PROJECT_SOURCE_DIR}/plugins/filters/bytedelta/bytedelta.c
+                PROPERTIES COMPILE_OPTIONS "-msse4.2")
```

Comment:
I think code as-is does not actually need SSE4.2 -- just SSSE3 should be enough (due to `_mm_shuffle_epi8` and `_mm_alignr_epi8` which are both SSSE3). So clang/gcc should be fine with just `-mssse3` flag, and MSVC does not actually need any special flag.

Reply:
You are right indeed.  Thanks!